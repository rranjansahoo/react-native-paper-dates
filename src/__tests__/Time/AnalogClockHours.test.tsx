import * as React from 'react'

import renderer from 'react-test-renderer'
import AnalogClockHours from '../../Time/AnalogClockHours'

it('renders AnalogClockHours', () => {
  const tree = renderer
    .create(<AnalogClockHours is24Hour hours={12} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
