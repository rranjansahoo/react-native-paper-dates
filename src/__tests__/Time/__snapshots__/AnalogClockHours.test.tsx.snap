// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AnalogClockHours 1`] = `
[
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 179,
          "top": 39,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        1
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 217,
          "top": 77,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        2
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 230,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        3
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 217,
          "top": 179,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        4
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 179,
          "top": 217,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        5
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 230,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        6
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 77,
          "top": 217,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        7
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 39,
          "top": 179,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        8
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 26,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        9
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 39,
          "top": 77,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        10
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 77,
          "top": 39,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        11
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 26,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              null,
            ],
          ]
        }
      >
        00
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 160,
          "top": 73,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        13
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 183,
          "top": 96,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        14
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 192,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        15
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 183,
          "top": 160,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        16
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 160,
          "top": 183,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        17
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 192,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        18
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 96,
          "top": 183,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        19
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 73,
          "top": 160,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        20
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 64,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        21
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 73,
          "top": 96,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        22
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 96,
          "top": 73,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                null,
              ],
            ],
          ]
        }
      >
        23
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 64,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              [
                {
                  "color": "rgba(255, 255, 255, 1)",
                },
              ],
            ],
          ]
        }
      >
        12
      </Text>
    </View>
  </View>,
]
`;
